src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -732,6 +732,17 @@ public ConsoleColorProxy(ConsoleHostUserInterface ui)
                 _ui = ui;
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-macos`
Write-Error Tests.ErrorRecord should not be truncated or have inserted newlines when redirected from another process
```powershell
Expected exactly 3, but got 0.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Error.Tests.ps1: line 106
106:         $result.Count | Should -BeExactly 3
```